File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int get_change(float dollars);

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Chash owed: ");
    }
    while(dollars < 0);
    
    printf("%i\n", get_change(dollars));
}
int get_change(float dollars)
{
    int cents = round(dollars * 100);
    
    int coins = 0;
    int denom[] = {25, 10, 5, 1};
    
    for (int i = 0; i < 4; i++)
    {
        coins += cents / denom[i];
        cents = cents % denom[i];
    }
    return coins;
}

File: cash.py
from cs50 import get_float
from sys import exit

def main():
    while True:
        dollars = get_float("Cash owed: ")
        if dollars >= 0:
            break
    cents = int(dollars * 100)
    print(change(cents))

def change(cents):
    return cents//25 +  (cents%25)//10 + (cents%25%10)//5 + cents%25%10%5

if __name__ == "__main__":
    main()